discordbot/index.js

```diff
@@ -66,31 +95,39 @@ setInterval(() => {
 
 // When the client is ready, run this code (only once)
 client.once('ready', () => {
-    getValueOfCurrenciesFrom("EUR");
-    client.user.setActivity('les actus, météo et autres', {type:"WATCHING", url:"https://oxey405.com/projects/monabot"});
+    setValueOfCurrenciesFrom("EUR");
+
+    // Set the client user's activity
+    client.user.setActivity('les actus, météo et autres', { type: "WATCHING"});
```

Comment:
Why deleting the URL ?

Reply:
URL is only use if the activity is being streamed, a link to the stream.
https://discord.js.org/#/docs/main/stable/class/Activity?scrollTo=url

Reply:
OK